ingest/one_vs_rest.csv

```diff
@@ -0,0 +1,41 @@
+,'type_0'_'rest'_qval,'type_0'_'rest'_log2fc,'type_0'_'rest'_mean,'type_1'_'rest'_qval,'type_1'_'rest'_log2fc,'type_1'_'rest'_mean,'type_2'_'rest'_qval,'type_2'_'rest'_log2fc,'type_2'_'rest'_mean,'type_3'_'rest'_qval,'type_3'_'rest'_log2fc,'type_3'_'rest'_mean

```

Comment:
Can we omit the single quotation marks in this row, and the other example data for user DE?  

It's reflective of the user's original data, but I suspect it represent be the majority of the data we'll ultimately get from users.  Having an automated test that we can handle group names enclosed in quotes would help ensure we keep supporting this likely less common case.